documentation/milestones/individual_milestones/static_site.md

```diff
@@ -0,0 +1,199 @@
+- As **maintainer of the project** I want:
+  - most of the critical development infrastructure to be in place when we officially launch the site, so that we can deploy bug fixes or new features quickly once the site is live.
+- As an **open source contributor**, I want:
+  - the site to link to resources like the repository, chat, etc., so that I can easily learn where and how to participate in the project.
+
+## Technical description
+
+### Infrastructure Requirements
+
+The infrastructure developed to deploy and host the site should balance:
+
+- Code quality
+- Security
+- Delivery velocity
+- Cost & maintenance
+
+### User Experience Requirements
+
+The design and structure of the site should balance:
+
+- Usability
+- Accessibility
+- Site performance
+- Brand identity
+
+### Content Requirements
+
+Process for drafting and updating the content of the site should balance:
+
+- Speed & ease of content management
+- Need for review and approval
+
+### Definition of done
+
+- [ ] The following infrastructure requirements are satisfied:
+  - [ ] The code needed to build and deploy the site is merged to `main`
+  - [ ] The site is built and hosted with the tools selected in the [Front-end Planning milestone](https://github.com/HHS/grants-equity/issues/49)
+  - [ ] All code quality checks set up in the [Developer Tools milestone](https://github.com/HHS/grants-equity/issues/50) are passing
+  - [ ] The resources required to deploy and host the site are provisioned programmatically using the framework established in the [Infrastructure-as-Code milestone](https://github.com/HHS/grants-equity/issues/123)
+  - [ ] Code changes are deployed using the CI/CD pipeline set up in [the Front-end CI/CD milestone](https://github.com/HHS/grants-equity/issues/58)
+- [ ] The following user experience (UX) requirements are satisfied:
+  - [ ] Anyone can access a live version of the site at beta.grants.gov
+  - [ ] The site adopts the UI principles and framework established in the [Foundational UI milestone](https://github.com/HHS/grants-equity/issues/60)
+  - [ ] The site is 508 compliant and satisfies the other guidelines outlined in the Accessibility Planning milestone
+- [ ] The following content requirements are satisfied:
+  - [ ] The site contains information about both the NOFO Transformation and Grants.gov Modernization workstreams
+  - [ ] The content on the site has been been reviewed and approved by the relevant stakeholders within each workstream
+  - [ ] The site also links to external resources related to the project (if they are available), including:
+    - [ ] The legacy grants.gov site
+    - [ ] The open source repository
+    - [ ] The open source wiki
+    - [ ] The open source chat
+  - [ ] Stretch goal: The site provides a mechanism to collect stakeholder feedback or questions
+
+### Proposed metrics for measuring goals/value/definition of done
```

Comment:
It might be worth making a note here to make sure there are tickets created and developer time dedicated to ensuring these metrics get tracked automatically and posted publicly automatically. This might look like writing scripts that pull data from the Google Analytics API, from lighthouse, from the build server, etc.

These can all happen *after* the initial deployment of beta.grants.gov to PROD, but we should make sure they do not get forgotten or missed. 

Tagging @sumiat for awareness.

Reply:
Added it in [this commit](https://github.com/HHS/grants-equity/pull/243/commits/b1620b74ecc0d590e7fc06575fd8f47edc035bc6)